kernel_hardening_checker/__init__.py

```diff
@@ -312,6 +339,10 @@ def main() -> None:
             # add relevant sysctl checks to the checklist
             add_sysctl_checks(config_checklist, arch)
 
+            arch, msg = detect_arch_sysctl(args.config, supported_archs)
```

Comment:
This function call is at the wrong place.

You can see this if you test the `kernel-hardening-checker` with `-c` and/or `-s`.

Reply:
yeah, i saw
i added call into Kconfig's if block...